app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt

```diff
@@ -3,31 +3,182 @@ package com.simply.birthdayapp.presentation.ui.screens.auth.signIn
+                                fontFamily = FontFamily(Font(R.font.karm_light)),
+                                fontSize = 16.sp
+                            )
+                        }
+                    }
+                    PasswordTextFiled(
+                        textState = password,
+                        label = stringResource(id = R.string.password),
+                        onValueChange = { input ->
+                            password = input
+                        }
+                    )
+                    Row(
+                        modifier = Modifier
+                            .wrapContentHeight()
+                            .fillMaxWidth(),
+                        Arrangement.SpaceBetween
+                    ) {
+                        Row(verticalAlignment = Alignment.CenterVertically) {
+                            Checkbox(
+                                checked = checkedState.value,
+                                onCheckedChange = { checkedState.value = it },
+                                colors = CheckboxDefaults.colors(
+                                    checkedColor = Primary1
+                                )
+                            )
+                            Text(
+                                text = stringResource(id = R.string.remember_password),
+                                fontSize = 12.sp,
+                                fontFamily = FontFamily(Font(R.font.karm_light)),
+                                fontWeight = FontWeight(700),
+                                color = Primary2
+                            )
+                        }
+                    }
+                    Row(
+                        modifier = Modifier
+                            .height(32.dp)
+                            .fillMaxWidth(),
+                        Arrangement.End
+                    ) {
+                        TextButton(onClick = onForgotPasswordClick) {
+                            Text(
+                                text = stringResource(id = R.string.forgot_password),
+                                fontSize = 12.sp,
+                                fontFamily = FontFamily(Font(R.font.karm_light)),
+                                fontWeight = FontWeight(700),
+                                color = Primary2
+                            )
+                        }
+                    }
+                    AuthButton(
+                        shape = RoundedCornerShape(24.dp),
+                        buttonTitle = stringResource(id = R.string.register),
+                        backgroundColor = Primary1,
+                        textColor = Primary2
+                    )
+                }
+            }
+        }
     }
 }
 
 @Preview(showBackground = true)
 @Composable
-private fun RegisterScreenPreview() {
-    SignInScreen()
+private fun SignInScreenPreview() {
+    SignInScreen(registerViewModel = get())
```

Comment:
Please use getViewModel() instead of get()

Reply:
Fixed